frontend/survey-project/src/app/shared/dto/requests/user/logout-request.ts

```diff
@@ -0,0 +1,3 @@
+export interface LogoutRequest {
```

Comment:
There is no need for LogoutRequest interface, authorization is set in header by intercept.